packages/kit/src/exports/vite/index.js

```diff
@@ -248,7 +251,10 @@ async function kit({ svelte_config }) {
 			is_build = config_env.command === 'build';
 
 			env = get_env(kit.env, vite_config_env.mode);
-
+			const emulator = await svelte_config.kit.adapter?.emulate?.();
+			//@ts-ignore
+			const platform_env = emulator?.platform?.(false)?.env;
+			wrangler_env = await get_wrangler_env(platform_env, kit.env);
```

Comment:
the core SvelteKit code shouldn't know anything about wrangler. anything platform-specific should be done through the adapters

Reply:
So the name of variable `wrangler_env` and function `get_wrangler_env` is the problem right. because we have platform env availabIe in `svelte_config.kit.adapter.emulate()` anyways. I will change it to 
```js
const emulator = await svelte_config.kit.adapter?.emulate?.();
//platform_env is global here
platform_env = split_env_var(emulator?.platform?.(false)?.env, kit.env);

```

is it ok?